soc/st/stm32/stm32wbax/hci_if/linklayer_plat_adapt.c

```diff
@@ -4,22 +4,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <zephyr/irq.h>
 #include <zephyr/kernel.h>
-#include <zephyr/arch/cpu.h>
-#include <zephyr/sys/util.h>
 #include <zephyr/drivers/entropy.h>
 #include <zephyr/logging/log.h>
```

Comment:
Are you sure file will compile w/o this ?

Reply:
Yes, It compiles